lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/OperatorTopology.cs

```diff
@@ -393,7 +392,7 @@ private void WaitForTaskRegistration(IList<string> identifiers, CancellationToke
 
                 var leftOver = string.Join(",", identifiers.Where(e => !foundList.Contains(e)));
                 Logger.Log(Level.Error, "For node {0}, cannot find registered parent/children: {1}.", _selfId, leftOver);
-                Exceptions.Throw(new RemotingException("Failed to find parent/children nodes in operator topology for node: " + _selfId), Logger);
+                Exceptions.Throw(new InvalidOperationException("Failed to find parent/children nodes in operator topology for node: " + _selfId), Logger);
```

Comment:
Going off the list of [Common Exceptions from the dotnet documentation](https://docs.microsoft.com/en-us/dotnet/standard/exceptions/), I chose `InvalidOperationException` because this seems like it's in an "invalide state".